presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -450,10 +455,10 @@ void PrestoServer::run() {
   }
 
   PRESTO_SHUTDOWN_LOG(INFO)
```

Comment:
Shall we only log io executor as we have already logged driver executor in other places? Thanks!

Reply:
Yeah this is the log for exchange http (io) executor